deploy_tools/cli.py

```diff
@@ -338,6 +354,37 @@ def call(
     click.echo(result)
 
 
+@main.command(
+    short_help="Generates an encrypted keystore file. Creates a new account if no private key is provided."
+)
+@keystore_file_save_option
+@private_key_option
+def generate_keystore(keystore_path: str, private_key: str):
+    if path.exists(keystore_path):
```

Comment:
Why do you have to do this check again manually, `click.Path(exists=False)` should do this for you, doesnt it?

Reply:
Unfortunately not. Setting this to `False` skip the existence test. That's what [the documentation](https://click.palletsprojects.com/en/5.x/api/#click.Path) states and what I observed, because I expected this in first place as well.

Reply:
I pushed this minor change. :heavy_check_mark: